spec/std/int_spec.cr

```diff
@@ -13,6 +13,15 @@ private macro it_converts_to_s(num, str, **opts)
   end
 end
 
+private macro pending_win32_converts_to_s(num, str, **opts)
+  pending_win32 {{ "converts #{num} to #{str}" }} do
+    num = {{ num }}
+    str = {{ str }}
+    num.to_s({{ opts.double_splat }}).should eq(str)
+    String.build { |io| num.to_s(io, {{ opts.double_splat }}) }.should eq(str)
+  end
+end
```

Comment:
I'm not happy about duplicating this spec helper just to mark it as pending on win32.

It would probably be better to wrap it in `{% if flag?(:win32) %}` at the callsite instead. It's just three calls.